RD-3069 Update documentation for Blueprint flow changes (#1960)
cloudify-cosmo/docs.getcloudify.org: content/working_with/console/widgets/blueprints.md

#### Uploading a Blueprint

##### Using marketplace

1. Click the **Upload** button.
2. In the menu, select **Upload from Marketplace** option.

> 3. In the Blueprint marketplace dialog, click **Upload** button on one of the blueprints shown in a table.

Reviewer: Let's be consistent with the naming - at least across the same file:

```suggestion
3. In the Blueprint Marketplace modal, click **Upload** button on one of the blueprints shown in a table.
```

The default visibility is "Tenant", and according to the logged-in user's permissions you can also choose other levels of [resource visibilities]({{< relref "working_with/manager/resource-visibility.md" >}}).<br>
6. Click **Upload**.

##### Using {{< param cfy_composer_name >}}

1. Click the **Upload** button.

> 2. In the menu click **Generate in the Composer** option.
> 3. {{< param cfy_composer_name >}} will be opened in a new tab and there you can create blueprint.

Reviewer: ```suggestion
2. In the menu, click **Generate in the Composer** option.
3. {{< param cfy_composer_name >}} will be opened in a new tab and there you can create a blueprint.
```

* `Refresh time interval` - The time interval in which the widget’s data will be refreshed, in seconds. Default: 10 seconds

> * `Enable click to drill down` - This option enables redirecting to the blueprint’s drill-down page upon clicking on a specific blueprint. Default: True

Reviewer: Let's use more user-friendly terminology:

```suggestion
* `Enable click to drill down` - This option enables redirecting to the blueprint’s drill-down page upon clicking on a specific blueprint. Default: Yes
```

> * `Display style` - Defines how the blueprints list should be displayed. Can be either Catalog or Table. The deployments status column is only available in list mode.  Default: Table

Reviewer: What does "The deployments status column is only available in list mode." mean?

Deployment count is visible in both - Catalog and Table view.
Blueprint status is visible in both - Catalog and Table view.

So I think it should be removed:

```suggestion
* `Display style` - Defines how the blueprints list should be displayed. Can be either Catalog or Table. Default: Table
```

> * `Show Composer options` - This options enables to show {{< param cfy_composer_name >}} options in menu and in the blueprints list. Default: false

Reviewer: Let's be more grammatically correct and use more user-friendly terminology:

```suggestion
* `Show Composer options` - This option allows to show {{< param cfy_composer_name >}} options in menu and in the blueprints list. Default: No
```

> * `Marketplace tabs` - Allow to edit tabs in Marketplace modal.

Reviewer: That will be hard to understand for a new user. I would extend it:

```suggestion
* `Marketplace tabs` - Allows to define multiple sources from which blueprints are taken to populate Blueprint Marketplace modal. User can define a name and URL for each tab.
```

> * `Marketplace display style` - Defines how the Blueprints Marketplace should be displayed. Can be either Catalog or Table.  Default: Table

Reviewer: ```suggestion
* `Marketplace display style` - Defines how the Blueprints Marketplace modal should be displayed. Can be either Catalog or Table.  Default: Table
```

> * `List of fields to show in the marketplace table` - Allow to change the list of visible columns in the Blueprint Marketplace. Works only when Marketplace display style is set to Table. Default: Name, Description.

Reviewer: ```suggestion
* `List of fields to show in the marketplace table` - Allow to change the list of visible columns in the Blueprint Marketplace modal. Works only when Marketplace display style is set to Table. Default: Name, Description.
```
